app/controllers/concerns/hyku_addons/users_controller_behavior.rb

```diff
@@ -12,7 +12,15 @@ def index
 
     def show
       @user = User.find_by(email: params[:email], display_profile: true)
-      render json: { status: 403, code: 'forbidden', message: t("errors.users_forbidden") } if @user.blank?
+      return render json: { status: 403, code: 'forbidden', message: t("errors.users_forbidden") } if @user.blank?
+
+      query_string = "(creator_tesim:\"*#{@user.email}*\") AND visibility_ssi:open"
+      @user_document_id_list = ActiveFedora::SolrService.get(query_string, rowy: 1_000_000)['response']['docs'].pluck('id')
```

Comment:
Is the argument supposed to be called `rowy`?

Reply:
I think it actually is but I will double check

Reply:
I thinks it's probably rows isn't it. Ill change it

Reply:
Can you give your PR's a read through before you give them to anyone else, just incase you have little things like this in them that you can catch and fix up :)